src/patternfly/components/JumpLinks/jump-links.scss

```diff
@@ -152,12 +152,15 @@
 
 // stylelint-disable
 .pf-c-jump-links__item {
-  &.pf-m-current {
+  &.pf-m-current > .pf-c-jump-links__link {
     --pf-c-jump-links__link--before--BorderTopWidth: var(--pf-c-jump-links__item--m-current__link--before--BorderTopWidth);
     --pf-c-jump-links__link--before--BorderLeftWidth: var(--pf-c-jump-links__item--m-current__link--before--BorderLeftWidth);
     --pf-c-jump-links__link--before--BorderColor: var(--pf-c-jump-links__item--m-current__link--before--BorderColor);
     --pf-c-jump-links__link-text--Color: var(--pf-c-jump-links__item--m-current__link-text--Color);
   }
+  &.pf-m-current > .pf-c-jump-links__list::before {
+    border-color: transparent;
```

Comment:
This does it! Although two things:

1. This only disables the border on nested lists when the parent item is current. While that fixes this particular issue, it would be great if we can just disable that border on any nested list.

2. We have a var for the list border - `--pf-c-jump-links__list--before--BorderColor`. You can manipulate that variable to change the border color as opposed to writing out a selector and targeting the `border-color` property. And you can redefine that variable on a parent selector as opposed to targeting the actual element it applies to, since css vars are inherited.

For example, if you have this code below - here's a [codepen](https://codepen.io/mcoker/pen/vYXOyZJ):

```
<style>
:root {
  --link-color: blue;
}

a {
  color: var(--link-color);
}
</style>

<p><a href="#">blue link</a></p>
<p class="foo"><a href="#">also a blue link</a></p>
```

And you want to change the link color to red when it's in `.foo`, you can write:

```
.foo {
  --link-color: red;
}
```

instead of having to write

```
.foo a {
  --link-color: red;
}
```

Here is a codepen for that - https://codepen.io/mcoker/pen/bGwdBRx

Reply:
Thanks! That makes sense.